### DISCUSSION TITLE
Mark as a false positive / Ignore

### DISCUSSION THREAD

**Discussion by tompalmer** [theophanemayaud/video-simili-duplicate-cleaner#122] [Ideas]
an additional option that will prevent a specific pair to be shown again until cache resets

**Comment by theophanemayaud** [comment]
Like this #99 ? It’s coming very soon, with the arm build for macOS !

**Comment by theophanemayaud** [comment]
Hey @tompalmer the new version does just this ! I hope it helps :)
